### DISCUSSION TITLE
Exclude certain flickr pictures or authors.

### DISCUSSION THREAD

**Discussion by somanynights** [mcguirepr89/BirdNET-Pi#784] [Ideas]
The picture that resolves to 'Galah' (Eolophus Roseicapilla) is a rainbow lorikeet that has been incorrectly labelled/tagged/titled in flickr. 

https://flickr.com/photos/36319610@N04/4892923285

Would be neat to be able to exclude certain incorrect pictures. 

Edit: whoops duplicate of https://github.com/mcguirepr89/BirdNET-Pi/discussions/700

**Comment by ehpersonal38** [comment]
I've just done a commit which will blacklist this image from ever appearing in the future. To have this apply to your system, be sure to update (Tools -> System Controls -> Update).
